tests/api/test_api_ui_collection_viewsets.py

```diff
@@ -32,7 +32,7 @@ def test_unset_certified(self):
         self.versions_api.unset_certified.return_value = {}
 
         response = self.client.delete(
-            f"/{API_PREFIX}/v3/_ui/collections/ansible/nginx/versions/1.2.3/certified/"
+            f"/{API_PREFIX}/v3/_ui/collections/ansible/nginx/1.2.3/certified/"
```

Comment:
Should this be:

``` python
 f"/{API_PREFIX}/v3/_ui/collections/ansible/nginx/versions/1.2.3/certified/"
```

Reply:
ansd/Or... bindings/openapi.yaml needs to provide a path for `{API_PREFIX}/v3/_ui/collections/{namespace}/{name}/versions/{version}/certified/` ?

Something seems out of whack but not sure what.